tests/test_non_ascii_search_string/test_non_ascii_search_string.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: UTF-8 -*-
+
+from dxr.testing import SingleFileTestCase
+
+
+class NonAsciiSearchStringTests(SingleFileTestCase):
```

Comment:
Since you're no longer DxrInstanceTestCase, all those other files (the dxr.config and the whole code folder) are unnecessary. In fact, this testcase could even merge into `test_non_ascii_paths` (which could be renamed `test_unicode` or something).